src/pages/Lending/hooks/useRepaymentLockData.tsx

```diff
@@ -73,11 +73,21 @@ export const useRepaymentLockData = ({ accountId, assetId }: UseLendingPositionD
     enabled: !!accountId && !!assetId,
   })
 
+  const isRepaymentLockQueryEnabled = useMemo(() => {
+    // We always need the LOANREPAYMENTMATURITY value from the mimir query as repaymentMaturity
+    if (!mimir) return false
+    // We need position data to calculate the repayment lock for a specific account's position
+    if (!!accountId && !!assetId) return !!position && !!blockHeight
```

Comment:
We probably want to react on `isPositionQuerySuccess` instead, as there may not be a position for the active `AccountId`, but the query will still be succesful and return `null`

Reply:
9de2e25f50